src/plugins/flow.js

```diff
@@ -878,11 +878,11 @@ export default function (instance) {
   });
 
   instance.extend("toAssignable", function (inner) {
-    return function (node) {
+    return function (node, isBinding) {
```

Comment:
We can probably use ...args here and pass that down right? (although I see there's only 2 params)